scripts/jina-hub-update.py

```diff
@@ -0,0 +1,77 @@
+""" Script to change versioning of files (eg. manifest.yml) for executors
+[encoders, crafters, indexers, rankers].
+It also adds the required jina version.
+Commits the change in the branch and raises a PR for the executor.
+"""
+import glob
+import os
+import subprocess
+import sys
+
+import git
+from ruamel.yaml import YAML
+
+repo_dir = os.getcwd()
+yaml = YAML()
+
+version_file = 'manifest.yml'
+
+
+def main():
+    hub_repo = git.Repo('jina/hub')
```

Comment:
the PR need to be made to `https://github.com/jina-ai/jina-hub/` and not to change directly the files under the submodule.

We will update the `jina-hub` repo and then jina will update its `hub` submodule folder but those are independent processes.

Reply:
Ah ok I see